### DISCUSSION TITLE
VDD更新了！

### DISCUSSION THREAD

**Discussion by NEANC** [ClassicOldSong/Apollo#167] [General]
o7，VDD已发布最新稳定版，请问有追踪上游的想法吗？
https://github.com/VirtualDisplay/Virtual-Display-Driver/releases/tag/24.12.24

**Comment by ClassicOldSong** [comment]
没有，这些更新一个月前我就看过了，对串流没有意义，甚至实用意义都很低

**Reply by ClassicOldSong** [reply]
具体解释在这里：

https://github.com/ClassicOldSong/Apollo/issues/164#issuecomment-2561076349

**Reply by NEANC** [reply]
> 没有，这些更新一个月前我就看过了，对串流没有意义，甚至实用意义都很低

好吧，看起来这个更新来得有些晚了:)
